### DISCUSSION TITLE
Calling Nova Toast from routes or controllers or tools

### DISCUSSION THREAD

**Discussion by RealityGamin** [laravel/nova-issues#5533] [Q&A]
Currently, the only two ways I know of that you can call Nova 4 toast message is thru:

1. Action::danger('message') - from Actions
2. Nova.error('message') - from Vue

Is there a way to do it from a Controller? or Route? or the Tools?

I have been trying things all day like this, and can't figure it out 
```
//From the nova::tool - inertia.php
Route::get('/', function (NovaRequest $request) {
    return redirect('/resources/users')->with('toast', [
        'type' => 'danger',
        'message' => "You cannot access this resource directly."
    ]);
});
```

I also tried `return Inertia::location('/resources/users')->with('message', 'This is a success message!');`

Thank you very much!

**Comment by crynobone** [comment]
Duplicate of https://github.com/laravel/nova-issues/discussions/5434
